Opening a form that uses the digest() function crashes the app

#### ODK Collect version
2026.4.0 beta

#### Problem description
Opening a form that uses the `digest()` XPath function crashes the app. It fails with:
```
Caused by: org.javarosa.xpath.XPathUnsupportedException: The problem was located in Calculate expression for /data/digest                                                                                                
  XPath evaluation: unsupported construct [digest('abc', 'SHA-256', 'hex')]                                                                                                                                                
      at org.javarosa.xpath.XPathConditional.evalRaw(XPathConditional.java:75)                                                                                                                                             
      at org.javarosa.core.model.condition.Recalculate.eval(Recalculate.java:50)                                                                                                                                           
      at org.javarosa.core.model.condition.Triggerable.apply(Triggerable.java:119)                                                                                                                                         
      at org.javarosa.core.model.QuickTriggerable.apply(QuickTriggerable.java:41)                                                                                                                                          
      at org.javarosa.core.model.TriggerableDag.evaluateTriggerable(TriggerableDag.java:566)
```

#### Steps to reproduce
1. Create a form with a calculate expression using `digest('abc', 'SHA-256', 'hex')` e.g. [digest.xlsx](https://github.com/user-attachments/files/29907857/digest.xlsx)
2. Install a release (minified) build of Collect.
3. Open the form.

#### Expected behavior
The form opens successfully, and `digest()` evaluates normally without crashing, matching the behavior of debug builds.

--- a/collect_app/proguard-rules.txt
+++ b/collect_app/proguard-rules.txt
@@ -21,6 +21,7 @@
 }
 -keep class org.javarosa.core.model.instance.geojson.GeojsonFeature { *; }
 -keep class org.javarosa.core.model.instance.geojson.GeojsonGeometry { *; }
+-keep class org.javarosa.xpath.expr.DigestAlgorithm { *; }
 
 # Make sure TaskSpecs are kept as they are instantiated via reflection
 -keep public class * implements org.odk.collect.async.TaskSpec { *; }

